WordPress/gutenberg
# Preformatted: Add spacing support

## What?

Adds margin and padding to the preformatted text block.

## Why?

For consistency, to allow users to add padding to the block.

## How?


## Testing Instructions
Make sure that the active theme supports spacing, for example in theme.json.

Create a new posts or page.
Add a preformatted text block.
Confirm that there is a dimension panel, and that the padding and margin settings are available but not enabled by default.

Test the padding and margin settings in the block editor, Site Editor Styles sidebar (global styles) and front.
Adjust the padding and margin via theme.json and confirm that it is applied in the editors and front.

## Screenshots or screencast 

https://user-images.githubusercontent.com/7422055/223356862-c51995d6-e875-4388-9efe-d911dcd7c019.mov
